app/customer.py

```diff
@@ -0,0 +1,53 @@
+import math
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list[int],
+            money: int,
+            car: dict
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+    def customer_has_money(self) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+    def cost_of_the_trip(
+            self,
+            fuel_prize: float,
+            customer_location: list[int],
+            shop_location: list[int]) -> float:
```

Comment:
Ensure that `customer_location` and `shop_location` are validated as lists of integers. This is important to prevent potential runtime errors if invalid data is passed to the method.